### PR TITLE
Ananya library

### PR DESCRIPTION
**Pull request by ananya-anni**
Connection with postgres database

### CODE REVIEW

File: LibrarySystemService/src/main/java/com/target/ready/library/system/service/LibrarySystemService/Entity/Book.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import lombok.Data;
+
+@Entity
+@Data
+@Table(name="Book")
+public class Book {
+    @Id
+    private int bookId;
+    private String bookName;
+    private String bookDesc;
```
Comment (akarsh7791) [inline LibrarySystemService/src/main/java/com/target/ready/library/system/service/LibrarySystemService/Entity/Book.java:15]:
```suggestion
    private String bookDescription;
```

File: LibrarySystemService/src/main/java/com/target/ready/library/system/service/LibrarySystemService/Repo/AuthorRepo.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package com.target.ready.library.system.service.LibrarySystemService.Repo;
+
+import com.target.ready.library.system.service.LibrarySystemService.Entity.Author;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface AuthorRepo extends JpaRepository<Author, Integer> {
```
Comment (akarsh7791) [inline LibrarySystemService/src/main/java/com/target/ready/library/system/service/LibrarySystemService/Repo/AuthorRepo.java:8]:
```suggestion
public interface AuthorRepository extends JpaRepository<Author, Integer> {
```

File: LibrarySystemService/src/main/java/com/target/ready/library/system/service/LibrarySystemService/Repo/BookRepo.java [hunk 3]
```diff
@@ -0,0 +1,9 @@
+package com.target.ready.library.system.service.LibrarySystemService.Repo;
+
+import com.target.ready.library.system.service.LibrarySystemService.Entity.Book;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface BookRepo extends JpaRepository<Book, Integer> {
```
Comment (akarsh7791) [inline LibrarySystemService/src/main/java/com/target/ready/library/system/service/LibrarySystemService/Repo/BookRepo.java:8]:
```suggestion
public interface BookRepository extends JpaRepository<Book, Integer> {
```

File: LibrarySystemService/src/main/java/com/target/ready/library/system/service/LibrarySystemService/Repo/CategoryRepo.java [hunk 4]
```diff
@@ -0,0 +1,9 @@
+package com.target.ready.library.system.service.LibrarySystemService.Repo;
+
+import com.target.ready.library.system.service.LibrarySystemService.Entity.Category;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface CategoryRepo extends JpaRepository<Category, Iterable> {
```
Comment (akarsh7791) [inline LibrarySystemService/src/main/java/com/target/ready/library/system/service/LibrarySystemService/Repo/CategoryRepo.java:8]:
```suggestion
public interface CategoryRepository extends JpaRepository<Category, Iterable> {
```

Comment (ananya-anni) [reply LibrarySystemService/src/main/java/com/target/ready/library/system/service/LibrarySystemService/Repo/CategoryRepo.java]: Committed all the changes

**Review by ISHAASTHANA — APPROVED** [review]
Looks fine as of now.

**Review by Jahnavi-pabbathi — APPROVED** [review]

**Pull request merged**